packages/admin-portal/src/utils/navigation.ts

```diff
@@ -0,0 +1,19 @@
+import { History } from 'history'
+import { MouseEvent } from 'react'
+
+export enum ExternalPages {
+  platformAPIDocs = 'https://foundations-documentation.reapit.cloud/api/api-documentation',
+}
+
+export const openNewPage = (uri: ExternalPages | string) => (event?: MouseEvent) => {
+  event?.preventDefault()
+  event?.stopPropagation()
+  window.open(uri, '_blank')
```

Comment:
if this is for third party websites we could add no opener so they don't have access to our page, and no referer so they don't know the user was previously on our page
```suggestion
  window.open(uri, '_blank', 'noopener,noreferrer')
```

Reply:
Updated, cool 👍🏻